pycqed/measurement/measurement_control.py

```diff
@@ -839,6 +841,30 @@ def save_MC_metadata(self, data_object=None, *args):
         set_grp.attrs['measurement_name'] = self.measurement_name
         set_grp.attrs['live_plot_enabled'] = self.live_plot_enabled()
 
+    @classmethod
+    def save_exp_metadata(self, metadata: dict, data_object):
+        '''
+        Saves experiment metadata to the data file.
```

Comment:
Maybe add a comment as to what the location in the data object is it get's saved to.